codegen/smithy-ruby-codegen-test/integration-specs/errors_spec.rb

```diff
@@ -10,6 +10,17 @@ module Errors
         error = ApiError.new(http_resp: http_resp, error_code: 'error')
         expect(error).to be_a(Hearth::HTTP::ApiError)
       end
+
+      it 'stores a request_id' do
```

Comment:
Does this still apply or should it be metadata?

Reply:
Fixed and deleted